tests/mock_network/accumulate.rs

```diff
@@ -37,7 +32,7 @@ fn messages_accumulate_with_quorum() {
     let dst = Authority::ManagedNode(nodes[0].name()); // The closest node.
                                                        // The smallest number such that
                                                        // `quorum * QUORUM_DENOMINATOR > section_size * QUORUM_NUMERATOR`:
-    let quorum = 1 + (section_size * QUORUM_NUMERATOR) / QUORUM_DENOMINATOR;
+    let quorum = 1 + section_size / 3;
```

Comment:
Should it use the constant?

Reply:
Ah, I missed this place. Yes, it probably should :+1: